srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -1,68 +1,92 @@
+"""
+This script takes converts one realisation CSV file to an SRF output file.
+
+The parameters that are read from the CSV file are documented on the
+[wiki](https://wiki.canterbury.ac.nz/display/QuakeCore/File+Formats+Used+On+GM).
+"""
+
 import argparse
+import os
+import subprocess
```

Comment:
Just use the subprocess, instead of importing the module and importing Popen separately?